src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -144,9 +144,13 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
     public List<DataAccessRequestManage> describeDataAccessRequestManageV2(AuthUser authUser) {
         List<DataAccessRequest> allDars = findAllDataAccessRequests();
         List<DataAccessRequest> filteredAccessList = dacService.filterDataAccessRequestsByDac(allDars, authUser);
-        filteredAccessList.sort(sortTimeComparator());
-        if (CollectionUtils.isNotEmpty(filteredAccessList)) {
-            return createAccessRequestManageV2(filteredAccessList);
+        List<DataAccessRequest> openDarList = filteredAccessList.stream().filter(dar -> {
+            DataAccessRequestData data = dar.getData();
+            return (Objects.isNull(data) ? true : (Objects.isNull(data.getStatus())));
```

Comment:
This filters on a null status, which isn't exactly what we're looking for. What we want is to filter out anything with a status of "Canceled". The semantics are different in that there could be a status we want to include and this would lead to some head-scratching as to why certain statuses were not showing up.